.rubocop.yml

```diff
@@ -1,80 +1,35 @@
-Metrics/MethodLength: { Max:  24 }
-Metrics/ClassLength:  { Max: 240 }
-Metrics/ModuleLength: { Max: 240 }
-Metrics/LineLength:   { Max: 112 }
-Metrics/CyclomaticComplexity: { Max: 8 }
-Metrics/PerceivedComplexity: { Max: 8 }
-Metrics/ParameterLists: { Max: 4 }
-Metrics/MethodLength: { Max: 24 }
-Metrics/AbcSize: { Max: 20 }
-
-
-Style/IndentHash: { EnforcedStyle: consistent }
-Style/HashSyntax: { EnforcedStyle: hash_rockets }
-Style/SignalException: { EnforcedStyle: only_raise }
-Style/AlignParameters: { EnforcedStyle: with_fixed_indentation }
-Style/StringLiteralsInInterpolation: { EnforcedStyle: double_quotes }
-Style/MultilineMethodCallIndentation: { EnforcedStyle: indented }
-Style/MultilineOperationIndentation: { EnforcedStyle: indented }
-Style/FirstParameterIndentation: { EnforcedStyle: consistent }
-Style/StringLiterals: { EnforcedStyle: double_quotes }
-Style/RegexpLiteral: { EnforcedStyle: slashes }
-Style/IndentArray: { EnforcedStyle: consistent }
-Style/ExtraSpacing: { AllowForAlignment: true }
-
-Style/PercentLiteralDelimiters:
-  PreferredDelimiters:
-    '%q': '{}'
-    '%Q': '{}'
-    '%r': '!!'
-    '%s': '()'
-    '%w': '()'
-    '%W': '()'
-    '%x': '()'
-
-Style/AlignArray:                     { Enabled: false }
-Style/StringLiterals:                 { Enabled: false }
-Style/Documentation:                  { Enabled: false }
-Style/DoubleNegation:                 { Enabled: false }
-Style/UnneededCapitalW:               { Enabled: false }
-Style/EmptyLinesAroundModuleBody:     { Enabled: false }
-Style/EmptyLinesAroundAccessModifier: { Enabled: false }
-Style/BracesAroundHashParameters:     { Enabled: false }
-Style/SpaceInsideBrackets:            { Enabled: false }
-Style/IfUnlessModifier:               { Enabled: false }
-Style/ModuleFunction:                 { Enabled: false }
-Style/RescueModifier:                 { Enabled: false }
-Style/GuardClause:                    { Enabled: false }
-Style/FileName:                       { Enabled: false }
-Lint/UselessAccessModifier:           { Enabled: false }
-Lint/HandleExceptions:                { Enabled: false }
-Style/SpaceAroundOperators:           { Enabled: false }
-Style/RedundantReturn:                { Enabled: false }
-Style/SingleLineMethods:              { Enabled: false }
+inherit_gem:
+  jekyll: .rubocop.yml
 
 AllCops:
-  TargetRubyVersion: 2.0
+  TargetRubyVersion: 2.2
   Include:
     - lib/**/*.rb
 
   Exclude:
-    - .rubocop.yml
-    - .codeclimate.yml
-    - .travis.yml
     - .gitignore
     - .rspec
-
+    - .rubocop.yml
+    - .travis.yml
+    - History.markdown
     - Gemfile.lock
-    - CHANGELOG.md
-    - readme.md
+    - LICENSE.txt
     - README.md
-    - Readme.md
-    - ReadMe.md
-    - COPYING
-    - LICENSE
-
-    - test/**/*
-    - vendor/**/*
```

Comment:
Looks like we need to exclude `vendor/**/*` from Rubocop

https://travis-ci.org/jekyll/jekyll-watch/jobs/289177802#L1103